mlflow/server/handlers.py

```diff
@@ -4222,4 +4665,29 @@ def _get_dataset_records_handler(dataset_id):
     ListScorerVersions: _list_scorer_versions,
     GetScorer: _get_scorer,
     DeleteScorer: _delete_scorer,
+    # Secrets APIs
+    CreateGatewaySecret: _create_secret,
+    GetGatewaySecretInfo: _get_secret_info,
+    UpdateGatewaySecret: _update_secret,
+    DeleteGatewaySecret: _delete_secret,
+    ListGatewaySecrets: _list_secrets,
+    # Endpoints APIs
+    CreateGatewayEndpoint: _create_endpoint,
+    GetGatewayEndpoint: _get_endpoint,
+    UpdateGatewayEndpoint: _update_endpoint,
+    DeleteGatewayEndpoint: _delete_endpoint,
+    ListGatewayEndpoints: _list_endpoints,
+    # Model Definitions APIs
+    CreateGatewayModelDefinition: _create_model_definition,
+    GetGatewayModelDefinition: _get_model_definition,
+    ListGatewayModelDefinitions: _list_model_definitions,
+    UpdateGatewayModelDefinition: _update_model_definition,
+    DeleteGatewayModelDefinition: _delete_model_definition,
+    # Endpoint Model Mappings APIs
+    AttachModelToGatewayEndpoint: _attach_model_to_endpoint,
+    DetachModelFromGatewayEndpoint: _detach_model_from_endpoint,
+    # Endpoint Bindings APIs
+    CreateGatewayEndpointBinding: _create_endpoint_binding,
+    DeleteGatewayEndpointBinding: _delete_endpoint_binding,
+    ListGatewayEndpointBindings: _list_endpoint_bindings,
```

Comment:
Shall we add proto for `get_resource_endpoint_configs`?

Reply:
^ same answer as preceding question